pubspec.yaml

```diff
@@ -14,13 +14,15 @@ web:
 dependencies:
   built_collection: ^1.0.0
   built_value: '>=2.0.0 <5.0.0'
+  # TODO: Remove in 3.0.0.
   dart_style: ^1.0.0
   matcher: ^0.12.0
   meta: ^1.0.5
 
 dev_dependencies:
   build_runner: '>=0.4.0 <0.6.0'
   built_value_generator: '>=2.0.0 <5.0.0'
+  dart_style: ^1.0.0
```

Comment:
I thought pub would yell if the same dependency shows up twice?

Reply:
You're right. Removed the dupe.